acrobat/blocks/app-banner/app-banner.js

```diff
@@ -0,0 +1,128 @@
+import { createTag } from "../../scripts/miloUtils.js";
+
+var link = '';
```

Comment:
We aren't really using `var` anymore. [Read more ](https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/)
There are few in this PR.

Reply:
Fixed.